src/wasm/encoder.cc

```diff
@@ -90,6 +90,13 @@ void WasmFunctionBuilder::AddBody(
   }
 }
 
+void WasmFunctionBuilder::AddBody(const byte instruction, bool is_local) {
```

Comment:
The name AddBody() is weird. You are starting an opcode, yes?
Can you use the WasmOpcode enum here?